### DISCUSSION TITLE
How to migrate TheHive data? need to allow access to elasticsearch port 9200

### DISCUSSION THREAD

**Discussion by glwallum** [Security-Onion-Solutions/securityonion#7450] [Unsupported Versions] [answered]
With TheHive leaving Security Onion, we are working on migrating our Hive data to a new instance. TheHive migration requires access to the Elasticsearch http://ES:9200 of the existing data.

What needs to be done to allow this access? I added a firewall-cmd rule to allow the traffic but it still is unreachable.

**Comment by weslambert** [comment, marked as answer]
If you are attempting to connect to TheHive's database to migrate the data, you'll want to connect to TheHive's Elasticsearch instance, which uses port 9400, not 9200.
